w5e3.py

```diff
@@ -0,0 +1,31 @@
+"w5e3"
```

Comment:
w5e3?

Reply:
Just the name of the exercise .. I deleted